src/main/components/DataTree.jsx

```diff
@@ -16,14 +17,20 @@ const DataTree = (props) => {
 
   useEffect(() => {
     const test = document.getElementById(`tree-${treeCount}`);
-    // renderTree(test);
   });
 
   return (
     <section className='wrapper' id={`tree-${treeCount}`} style= {{ maxHeight: '350px', overflow: 'auto' }} >
       {/* Tree gets rendered here after component mounts */}
-      <ReactJson src={data} theme='bright:inverted' iconStyle='square'
-        onEdit={changeObject} onAdd={changeObject} onDelete={changeObject}/>
+      <ReactJson 
```

Comment:
:fire:

Reply:
Wait… that’t not actually the right version 🤔